modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/GoServerCodegen.java

```diff
@@ -31,16 +32,26 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(GoServerCodegen.class);
 
-    protected String apiVersion = "1.0.0";
-    protected int serverPort = 8080;
+    protected String packageVersion = "1.0.0";
+    protected String serverPort = "8080";
```

Comment:
@mapero Thanks for the contribution. should the port number be an integer instead?